commerce-account-web/src/main/java/com/liferay/commerce/account/web/internal/frontend/CommerceAccountAddressClayTable.java

```diff
@@ -92,27 +92,25 @@
 
 		Address address = (Address)model;
 
-		StringBundler sb = new StringBundler(7);
+		StringBundler sb = new StringBundler(3);
 
-		sb.append("javascript:deleteCommerceAddress");
-		sb.append(StringPool.OPEN_PARENTHESIS);
-		sb.append(StringPool.APOSTROPHE);
+		sb.append("deleteCommerceAddress('");
 		sb.append(address.getAddressId());
-		sb.append(StringPool.APOSTROPHE);
-		sb.append(StringPool.CLOSE_PARENTHESIS);
-		sb.append(StringPool.SEMICOLON);
+		sb.append("');");
```

Comment:
This would ideally still be 

    sb.append(StringPool.APOSTROPHE);
    sb.append(StringPool.CLOSE_PARENTHESIS);
    sb.append(StringPool.SEMICOLON);

-- just combine `StringPool` elements into string literals if there's already a string literal being concatenated adjacent...ly.